README.md

```diff
@@ -20,6 +20,7 @@ The intended area of it's usage is data mining.
 ```
 export GO111MODULE=on
 go mod download
+make
```

Comment:
The bindata is already generated, this should not be required. `make` generates it if it does not exist.

Reply:
I found that the CI complains 

`flag provided but not defined: -nometadata` 
https://travis-ci.com/src-d/go-license-detector/jobs/233963633

So I thought an older version of `bindata` was used, but it seems not.

Reply:
OK so let's re-generate bindata in the CI so that we test `make`, but remove it from the README.

Reply:
The PR started getting messy, I squashed some commits.

I noted the detail about old version of `go-bindata` [here](https://github.com/src-d/go-license-detector/pull/61/commits/84eefa098e178c29934258b767412eebafa48563)